Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/ui/sheets/location/ContactDiaryLocationBottomSheetDialogViewModel.kt

```diff
@@ -21,22 +21,33 @@ class ContactDiaryLocationBottomSheetDialogViewModel @AssistedInject constructor
     @Assisted private val addedAt: String?,
     private val contactDiaryRepository: ContactDiaryRepository
 ) : CWAViewModel(dispatcherProvider = dispatcherProvider) {
+
     private val text = MutableStateFlow("")
 
     val isValid = text.map {
-        it.isNotEmpty() && it.length <= MAX_LOCATION_NAME_LENGTH
+        it.isNotEmpty()
     }.asLiveData()
 
     val shouldClose = SingleLiveEvent<Unit>()
 
+    private val formattedName: String
+        get() {
+            var newName = text.value
```

Comment:
`val newName = if (text.value.isNotBlank()) text.value.trim() else text.value`
to avoid var

Reply:
I would rather not because text.value can be changed from outside while this is executed